### PR TITLE
feat: optimize images for liveness blog post

### PR DESCRIPTION
**Pull request by 0xArdy**

### CODE REVIEW

File: public/blog-posts/liveness-2-and-beyond.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Let’s first recap the problem we are trying to solve with the Liveness Module.
...
 
-![https://defi-wonderland.notion.site/image/https%3A%2F%2Fprod-files-secure.s3.us-west-2.amazonaws.com%2F7683bccd-1174-4689-a817-b27fd9d7ef00%2Fda7cbc59-85ab-48bb-97c5-bc043bb6fd3e%2FDALLE_2023-11-01_15.45.54_-_Create_an_epic_and_dark_illustration_featuring_Alice_from_Alice_in_Wonderland_actively_engaged_in_the_process_of_synchronizing_two_different_worlds._.png?table=block&id=29965bf6-c267-441e-bec6-41ca29641920&spaceId=7683bccd-1174-4689-a817-b27fd9d7ef00&width=2000&userId=&cache=v2](https://defi-wonderland.notion.site/image/https%3A%2F%2Fprod-files-secure.s3.us-west-2.amazonaws.com%2F7683bccd-1174-4689-a817-b27fd9d7ef00%2Fda7cbc59-85ab-48bb-97c5-bc043bb6fd3e%2FDALLE_2023-11-01_15.45.54_-_Create_an_epic_and_dark_illustration_featuring_Alice_from_Alice_in_Wonderland_actively_engaged_in_the_process_of_synchronizing_two_different_worlds._.png?table=block&id=29965bf6-c267-441e-bec6-41ca29641920&spaceId=7683bccd-1174-4689-a817-b27fd9d7ef00&width=2000&userId=&cache=v2)
+![Create_an_epic_and_dark_illustration_featuring_Alice_from_Alice_in_Wonderland_actively_engaged_in_the_process_of_synchronizing_two_different_worlds._.png](src/assets/blogs/alice-lq.jpg)
```
Comment (0xGorilla) [inline public/blog-posts/liveness-2-and-beyond.md:13]: could we have them under `src/assets/blog-posts/liveness-2-and-beyond`?

Comment (0xArdy) [reply public/blog-posts/liveness-2-and-beyond.md:13]: sure

Comment (0xArdy) [reply public/blog-posts/liveness-2-and-beyond.md]: done https://github.com/defi-wonderland/web/pull/58/commits/ad3c96d72b1edfb79b1590737ce26db13f151229

File: public/blog-posts/liveness-2-and-beyond.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ Let’s first recap the problem we are trying to solve with the Liveness Module.
```
Comment (0xGorilla) [inline public/blog-posts/liveness-2-and-beyond.md:10]: image
image

These ones didn't change, am I right?

File: public/blog-posts/liveness-2-and-beyond.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ Let’s first recap the problem we are trying to solve with the Liveness Module.
 
 Unlike EOAs, smart wallets have configuration settings, which can cause synchronization problems across chains. Consequently, SAFEs on different chains function as separate contracts, even though they may share the same address and configuration parameters during deployment. This problem becomes critical when there’s a change in the owners’ list. We must tackle this issue if we want to create a chain-agnostic future.
```
Comment (0xGorilla) [inline public/blog-posts/liveness-2-and-beyond.md:11]: Also, the preview is not showing the images

File: public/blog-posts/liveness-2-and-beyond.md (hunk 2 above)
Comment (0xArdy) [reply public/blog-posts/liveness-2-and-beyond.md:10]: solved! https://github.com/defi-wonderland/web/pull/58/commits/7acb5b37855600479acb8fba03f0c6dba30805b6

File: public/blog-posts/liveness-2-and-beyond.md (hunk 3 above)
Comment (0xArdy) [reply public/blog-posts/liveness-2-and-beyond.md:11]: fixed https://github.com/defi-wonderland/web/pull/58/commits/7acb5b37855600479acb8fba03f0c6dba30805b6

**Review by 0xGorilla — APPROVED** [review]

**Comment by turtlemoji** [comment]
idem about img compression and quality

**Pull request merged**